fuzz/src/chanmon_consistency.rs

```diff
@@ -244,6 +244,7 @@ fn check_api_err(api_err: APIError) {
 				_ if err.starts_with("Cannot send value that would put counterparty balance under holder-announced channel reserve value") => {},
 				_ if err.starts_with("Cannot send value that would overdraw remaining funds.") => {},
 				_ if err.starts_with("Cannot send value that would not leave enough to pay for fees.") => {},
+				_ if err.starts_with("Cannot send value that would put our exposure to dust HTLC at") => {},
```

Comment:
Note that you added an `s` to HTLC in the latest push, so the check here needs to be updated to match.